courses/free-courses-ta.md

```diff
@@ -0,0 +1,62 @@
+### Index
+
+* [Android](#android)
+* [Bash and Shell](#bash-and-shell)
+* [C](#c)
+* [C++](#cpp)
+* [HTML and CSS](#html-and-css)
+* [Java](#java)
+* [JavaScript](#javascript)
+* [PHP](#php)
+* [Python](#python)
+
+
+### Android
+
+* [Android In Tamil 2019](https://youtube.com/playlist?list=PL4unWLKFsZfcGBja19mrwodNm6AyzZk2B) - Tutor Joe's Stanley
+
+
+### Bash and Shell
+
+* [Bash scripting in Tamil](https://youtube.com/playlist?list=PLgWpUXNR_WCeWiXmsYf5HUe7E4I29zTJr) - Payilagam
+
+
+### C
+
+* [C Tutorial in Tamil](https://youtube.com/playlist?list=PLBQXOA5OR76rq-sU8mNsUmj2Z9kQXz7oD) - Tamil Pro Techniques
+* [Fundamentals of C programming in Tamil](https://youtube.com/playlist?list=PLmjuBlzAWCzxTdRxTtMSj1NFe_GvFlvFw) - Collectiva Knowledge Academy
+
+
+### <a id="cpp"></a>C++
+
+* [C++ Programming in Tamil](https://youtube.com/playlist?list=PLWbtDrDnmTHBPD-Dt5BJi7iP11x6UvgU0) - CS in Tamil
+
+
+### HTML and CSS
+
+* [CSS in Tamil](https://youtube.com/playlist?list=PL73Obo20O_7gGv4cLEOoqTF8_m8rPKyQh) - CyberDude Networks Pvt. Ltd.
+* [HTML Course in Tamil](https://youtube.com/playlist?list=PL73Obo20O_7gcXt0cfQA14jey8zavtKAq) - CyberDude Networks Pvt. Ltd.
+* [Learn HTML in Tamil | HTML Tutorials in Tamil | Beginner to Advanced](https://youtube.com/playlist?list=PLpYn3LR7eQI2trAr1z1lmvYGAQfsbllY5) - Tamil Developer
```

Comment:
Do a Markdown escape for pipes present in resource titles (see #5176 for a informative reason)

```suggestion
* [Learn HTML in Tamil \| HTML Tutorials in Tamil \| Beginner to Advanced](https://youtube.com/playlist?list=PLpYn3LR7eQI2trAr1z1lmvYGAQfsbllY5) - Tamil Developer
```

Reply:
Resolved by commit a0e2f8203e6c91cc2fea737a703a653e2133e166